src/main/java/org/example/Main.java

```diff
@@ -1,22 +1,31 @@
 package org.example;
 import java.util.Scanner;
 
-import static java.lang.System.in;
-import static java.lang.System.out;
+import static java.lang.System.*;
 
 public class Main {
 
     public static final String CORRECT_NUMBER = "5";
+    public static  int MAX_TRIES = 5;
+    public static int NUMBER_TRIES= 0;
 
     public static void main(String[] args) {
-        Scanner number = new Scanner(in);
-        out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
-        String name = number.nextLine();
-        out.println(name);
-        if (CORRECT_NUMBER.equals(name)) {
-            out.println("Wszedles");
-        } else {
-            out.println("Nie prawidlowe haslo");
+        while (NUMBER_TRIES < MAX_TRIES) {
+            Scanner number = new Scanner(in);
+            out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
+            String name = number.nextLine();
+            out.println(name);
+            if (CORRECT_NUMBER.equals(name)) {
+                out.println("Wszedles");
+                break;
+            } else {
+                NUMBER_TRIES++;
+                out.println("Nie prawidlowe haslo");
+            }
```

Comment:
with single if and no other cases you don't have to use else - else is implicit.
Also, instead of using positive case you could use negative case and continue the loop instead of breaking it - so called "guard clauses"
Read about it here:
https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a